cloudformation/launchdetector.yaml

```diff
@@ -244,22 +259,29 @@ Resources:
       SecurityGroups:
       - !Ref InstanceSecurityGroup
       UserData: !Base64
-        "Fn::Sub":
-        - |
+        Fn::Sub: |
           #!/bin/bash -e
 
           mkdir -p /tmp/install
 
           aws s3 cp s3://gnm-multimedia-rr-deployables/${Stack}/${Stage}/${App}/multimedia-launchdetector-v3_3.0_all.deb /tmp/install
-          aws s3 cp s3://gnm-multimedia-rr-deployables/filebeat-6.1.2-amd64.deb /tmp/install
-          aws s3 cp s3://gnm-multimedia-rr-deployables/metricbeat-6.1.2-amd64.deb /tmp/install
-          dpkg -i /tmp/install/filebeat-6.1.2-amd64.deb
-          dpkg -i /tmp/install/metricbeat-6.1.2-amd64.deb
 
           declare -x MOST_RECENT_PACKAGE=`ls -Frt /tmp/install/*.deb | grep "[^/]$" | tail -n 1`
           echo Going to install $MOST_RECENT_PACKAGE
           dpkg --install "${!MOST_RECENT_PACKAGE}"
 
+          echo --------------------------------------
+          echo Updating DNS resolvers...
+          echo --------------------------------------
+          if [ "${ExtraDNS}" != "" ]; then
```

Comment:
this seems not to be the recommended way on systemd, but it works, which is more than i can say for any other method i have tried.  Since these are isolated instances not shared with anything else i am keeping this config for the time being (though it would be nicer to make a more systemd-ish config)